server/utils/response.go

```diff
@@ -31,8 +32,9 @@ func RespondWithError(w http.ResponseWriter, message string, status int) {
 // Reusable success response function
 func RespondWithSuccess(w http.ResponseWriter, status int, data interface{}) {
 	response := successResp{
-		Success: true,
-		Data:    data,
+		Success:   true,
+		Data:      data,
+		TimeStamp: time.Now().UTC().UnixNano(),
```

Comment:
time.Now().UnixNano() is good enough, no need to add UTC(). Also UnixNano() is in nano seconds, not milliseconds